lib/kits/events/tracker.ts

```diff
@@ -351,70 +394,16 @@ class Tracker {
             )
         );
     }
-}
-
-export default Tracker;
 
-let dispatching = false;
-const dispatchLimit = 100;
-
-let dispatchRetryInterval: NodeJS.Timeout | null | void = null;
-const dispatch = throttle(
-    async (
-        eventsTrackUrl: string,
-        eventsTrackHeaders: Record<string, string>
-    ) => {
-        if (!pool) {
-            console.warn('Tracker: dispatch called with no active event pool.');
-            return;
+    dispatching = false;
+    dispatchLimit = 100;
+    dispatch = throttle(this.dispatchBeacon, 4000);
```

Comment:
Going to swap this `throttle` for a more suitable `debounce`, such that we don't do a lot of immediate single-event dispatches.